Tests/test_file_container.py

```diff
@@ -16,7 +16,7 @@ def test_isatty(self):
         im = hopper()
         container = ContainerIO.ContainerIO(im, 0, 0)
 
-        self.assertEqual(container.isatty(), 0)
+        self.assertIs(container.isatty(), False)
```

Comment:
I know `assertIs` also checks the value is `False` by identity, but perhaps `assertFalse` is good enough here?

```suggestion
        self.assertFalse(container.isatty())
```

Reply:
Switched to `assertFalse` in the latest revision.